recipes/cpython/all/test_package/test_package.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+import os
+import sys
+
```

Comment:
```suggestion

# Hack to work around Python 3.8+ secure dll loading
# See https://docs.python.org/3/whatsnew/3.8.html#bpo-36085-whatsnew
if hasattr(os, "add_dll_directory"):
    for directory in os.environ.get("PATH", "").split(os.pathsep):
        if os.path.isdir(directory):
            os.add_dll_directory(directory)
```
Fixes the test package when dependencies are dynamically linked.

Reply:
Thanks!
This works perfectly!